tests/fixtures/dos_multiple_briefs_fixture.json

```diff
@@ -3,6 +3,7 @@
     {
       "applicationsClosedAt":"2016-03-24T00:00:00.000000Z",
       "backgroundInformation":"Here is the first requirements summary.",
+      "specialistRole":"Content Specialist",
```

Comment:
This is a brief for the user research participants lot. We only have specialist roles for the digital specialists lot so we don't want this key in here for this brief